src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs

```diff
@@ -41,13 +41,13 @@ public static class MilvusBuilderExtensions
     /// <returns>A reference to the <see cref="IResourceBuilder{MilvusServerResource}"/>.</returns>
     public static IResourceBuilder<MilvusServerResource> AddMilvus(this IDistributedApplicationBuilder builder,
         string name,
-        IResourceBuilder<ParameterResource> apiKey,
+        IResourceBuilder<ParameterResource>? apiKey = null,
         int? grpcPort = null)
     {
-        ArgumentNullException.ThrowIfNull(apiKey, nameof(apiKey));
+        var apiKeyParameter = apiKey?.Resource ??
+            ParameterResourceBuilderExtensions.CreateDefaultPasswordParameter(builder, $"{name}-Key", special: false);
```

Comment:
Why `special: false`?

Reply:
(super nit): in other places we use lower case names: `$"{name}-password"` but here we have PascalCase `-Key`. Should we be consistent?